import_export/utils.py

```diff
@@ -23,3 +27,61 @@ def __enter__(self):
     def __exit__(self, *args):
         if self.using_transactions:
             self.context_manager.__exit__(*args)
+
+
+def diff_lines_to_words(text1, text2):
+    # Implemented following the docs:
+    # https://github.com/google/diff-match-patch/wiki/Line-or-Word-Diffs
+    # Only one line is different from diff_linesToChars.
+    # line_end = text.find('\n', lineStart)
+    # --->
+    # line_end = text.find(' ', lineStart)
+    # Variable names are also changed to snake_case for code integrity.
+
+    line_array = []
+    line_hash = {}
+    line_array.append('')
+
+    def diff_lines_to_chars_munge(text):
+        chars = []
+        line_start = 0
+        line_end = -1
+        while line_end < len(text) - 1:
+            line_end = text.find(' ', line_start)
+            if line_end == -1:
+                line_end = len(text) - 1
+            line = text[line_start:line_end + 1]
+
+            if line in line_hash:
+                chars.append(chr(line_hash[line]))
+            else:
+                if len(line_array) == maxLines:
+                    line = text[line_start:]
+                    line_end = len(text)
+                line_array.append(line)
+                line_hash[line] = len(line_array) - 1
+                chars.append(chr(len(line_array) - 1))
+            line_start = line_end + 1
+        return "".join(chars)
+
+    maxLines = 666666
```

Comment:
Out of curiosity, where do these max lines values come from?

Reply:
I just copied it from diff_match_patch source code and changed a line. This is the doc that I removed: 

> Allocate 2/3rds of the space for text1, the rest for text2.

I think it is some optimization trick. I really don't understand it myself. And sorry for snakeCase. I didn't catch it. It will be fixed in the next commit.